backend/src/test/java/com/htecgroup/skynest/service/impl/UserServiceImplTest.java

```diff
@@ -114,17 +113,6 @@ void registerUser_AlreadyExists() {
     Assertions.assertThrows(UserException.class, () -> userService.registerUser(newUserDto));
```

Comment:
Maybe also add a verification that corresponding mock services are called and that the return error message is as expected